sdk/tests/iot/hub/test_az_iot_hub_client_twin.c

```diff
@@ -320,6 +322,24 @@ test_az_iot_hub_client_twin_parse_received_topic_reported_props_no_version_found
       response.response_type, AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_REPORTED_PROPERTIES);
 }
 
+static void test_az_iot_hub_client_twin_parse_received_topic_request_error_succeed()
+{
+  az_iot_hub_client client;
+  assert_int_equal(
+      az_iot_hub_client_init(&client, test_device_hostname, test_device_id, NULL), AZ_OK);
+  az_iot_hub_client_twin_response response;
+
+  assert_int_equal(
+      az_iot_hub_client_twin_parse_received_topic(
+          &client, test_twin_received_topic_bad_request_success, &response),
+      AZ_OK);
+
+  assert_true(az_span_is_content_equal(response.request_id, test_device_request_id));
+  assert_true(az_span_is_content_equal(response.version, AZ_SPAN_EMPTY));
+  assert_true(response.status >= AZ_IOT_STATUS_BAD_REQUEST);
```

Comment:
Shouldn't this be exactly equal?

Reply:
Well, it could for the sake of the string and the limited scope of the unit test, but the code itself covers more than just the 400 case.  I was debating keeping a 400 case and perhaps adding a second 400+ case?  Separate the => into = and>  Perhaps I should do just that.